### DISCUSSION TITLE
是否支持同步下载百度网盘指定文件夹到本地呢？

### DISCUSSION THREAD

**Discussion by Calypso723** [PeterDing/BaiduPCS-Py#100] [Q&A]
BaiduPCS-Py的下载速度很快，是我使用过的几个命令行客户端中最快的。
因为看到有“同步本地目录到远端”到远端的指令，所以想请问作者是否支持同步百度网盘指定文件夹到本地呢？
如果能有定时/实时同步下载功能就更棒了！

**Comment by PeterDing** [comment]
由于下面的百度网盘服务端的问题，“远端同步到本地”目前不好实现。
1. 远端文件md5是加密过的，无法对比本地文件。要得到正确的md5，需要发起下载请求，并且有时候也拿不到md5。
2. 远端文件的创建和更改时间会随不同客户端的操作发生改变。这会使本地的文件的修改被覆盖。

再加上，有加密文件的问题，就更不好处理了。

**Reply by Calypso723** [reply]
好的，非常感谢！
